packages/webapp/src/pages/election/index.tsx

```diff
@@ -64,6 +88,22 @@ export const ElectionPage = (props: Props) => {
             currentElection.election_seeder.end_time) ||
             currentElection.start_time);
 
+    const loggedInMemberName = currentMember?.name || accountName;
+    const loggedInMember: MemberData | undefined =
+        members &&
+        loggedInMemberName &&
+        getMemberRecordFromName(members, loggedInMemberName);
+
+    const RSVPStatus = [
+        "no_donation",
+        "in_election",
+        "not_in_election",
+        "recently_inducted",
+    ];
```

Comment:
What would you think about turning this into a [numeric enum](https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums)? Then we can reference it in this file, and your fixtures can also use it, instead of using magic numbers?